cfme/tests/configure/test_access_control.py

```diff
@@ -701,3 +701,35 @@ def test_superadmin_tenant_crud():
     with update(tenant):
         tenant.name = tenant.name + "edited"
     tenant.delete()
+
+
+@pytest.mark.uncollectif(lambda: version.current_version() < "5.5")
+def test_superadmin_tenant_project_crud():
+    """Test suppose to verify CRUD operations for CFME projects
+
+    Prerequisities:
+        * This test is not depending on any other test and can be executed against fresh appliance.
+
+    Steps:
+        * Create tenant
+        * Create project as child to tenant
+        * Update description of project
+        * Update name of project
+        * Delete project
+        * Delete tenant
+    """
+    tenant = ac.Tenant(
+        name='tenant1' + fauxfactory.gen_alphanumeric(),
+        description='tenant1 description')
+    tenant.create()
```

Comment:
```python
tenant.create()
request.addfinalizer(tenant.delete)
```

dtto for the project. For that you need to pass the `request` parameter to this function.